app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -391,7 +412,10 @@ def process_enrollment_response(enrollment, response)
 
     case response['status']
     when IPP_STATUS_PASSED
-      if SUPPORTED_ID_TYPES.include?(response['primaryIdType'])
+      if enrollment.capture_secondary_id_enabled &&
```

Comment:
Do you need both parts of this if check? Shouldn't it be sufficient to just check if the secondaryIdType is not supported or if secondary_capture_id_enabled is true?

Also, if secondary_capture_id_enabled is true, then that means here that we're not supporting secondary id's. Just noting that maybe the feature flag wording is a little confusing?

Reply:
good question - the reason for both checks here is to identify enrollments that were passed by usps using a secondary id while dav was enabled for that user. Since we don't want them to pass using the secondary id we'll change the status to "failed" even though the clerk marked the user as "passed". 
I did wonder if it might be better to change the check of the secondaryIdType to `enrollment.current_address_matches_id == false`.

Reply:
Both `capture_secondary_id_enabled` and `response['secondaryIdType']` need to be checked in order to determine whether the user gets retroactively failed by this logic.